packages/apps/graph-client/src/pages/account/overview/[module]/[account]/[chain].tsx

```diff
@@ -0,0 +1,142 @@
+import { Box, Grid, Notification, Table } from '@kadena/react-ui';
+
+import { useGetChainAccountQuery } from '../../../../../__generated__/sdk';
+import { CompactTransactionsTable } from '../../../../../components/compact-transactions-table/compact-transactions-table';
+import { CompactTransfersTable } from '../../../../../components/compact-transfers-table/compact-transfers-table';
+import Loader from '../../../../../components/loader/loader';
+import { mainStyle } from '../../../../../components/main/styles.css';
+import { Text } from '../../../../../components/text';
+
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+const ChainAccount: React.FC = () => {
+  const router = useRouter();
+
+  const {
+    loading: loadingChainAccount,
+    data: chainAccountQuery,
+    error,
+  } = useGetChainAccountQuery({
+    variables: {
+      moduleName: router.query.module as string,
+      accountName: router.query.account as string,
+      chainId: router.query.chain as string,
+    },
+  });
+
+  return (
+    <div style={{ padding: '0 50px 30px 50px' }}>
+      <Head>
+        <title>Kadena Graph Client</title>
+        <link rel="icon" href="/favicon.png" />
+      </Head>
+
+      <main className={mainStyle}>
+        <Text
+          as="h1"
+          css={{ display: 'block', color: '$mauve12', fontSize: 48, my: '$12' }}
+        >
+          Kadena Graph Client
+        </Text>
+
+        <div>
+          {loadingChainAccount && (
+            <div style={{ display: 'flex', alignItems: 'center' }}>
+              <Loader /> <span>Retrieving account information...</span>
+            </div>
+          )}
+          {error && (
+            <Notification.Root color="negative" icon="Close">
+              Unknown error:
+              <br />
+              <br />
```

Comment:
Instead of `<br>` maybe replace by `<Box>` here, like its done on other parts where spacing is needed

Reply:
Fixed!